ods_ci/tests/Resources/CLI/TrustyAI/trustyai_service.resource

```diff
@@ -0,0 +1,38 @@
+*** Settings ***
+Documentation     Collection of keywords to interact with TrustyAI service via CLI
+Resource          ../../../Resources/OCP.resource
+
+
+*** Variables ***
+${TRUSTYAI_RESOURCEPATH}=            tests/Resources/Files/TrustyAI
+${TRUSTYAI_CR_FILEPATH}=             ${TRUSTYAI_RESOURCEPATH}/trustyai_cr.yaml
+
+
+*** Keywords ***
+Install And Verify TrustyAI Service
+    [Documentation]    Install TrustyAI service CRD and verify that TrustyAI resources have spun up
+    [Arguments]        ${namespace}
+    ${return_code}    ${output}=    Run And Return Rc And Output    oc apply -f ${TRUSTYAI_CR_FILEPATH} -n ${namespace}
+    Sleep    60s    msg=Wait for Trusty Route to be created
+    ${return_code}    ${output}    Run And Return Rc And Output   oc get route trustyai-service -n ${namespace} --template={{.spec.host}}
+    Should Be Equal As Integers    ${return_code}	 0
+    RETURN       https://${output}
+
+Verify Model Is Registered With TrustyAI Service
+    [Documentation]    Verify the deployed model is registered with TrustyAI Service by verifying MM_PAYLOAD_PROCESSORS
+    ...                value is the TrustyAI service route
+    [Arguments]        ${namespace}=${PRJ_TITLE}
+    ${MM_PAYLOAD_PROCESSORS_Expected}=    Set Variable    http://trustyai-service.${namespace}.svc.cluster.local/consumer/kserve/v2
+    Wait Until Keyword Succeeds  3 min  20 sec  Verify Model Serving Pod Exists   ${namespace}    label_selector=modelmesh-service=modelmesh-serving
+    ${podname}=    Get Pod Name   ${namespace}    label_selector=modelmesh-service=modelmesh-serving
+    Log    Serving Runtime Podname: ${podname}
+    ${MM_PAYLOAD_PROCESSORS_Actual}=  Run  oc get pod ${podname} -n ${namespace} -o json | jq '.spec.containers[0].env[] | select(.name=="MM_PAYLOAD_PROCESSORS") | .value'
+    ${MM_PAYLOAD_PROCESSORS_Actual}=    Strip String    ${MM_PAYLOAD_PROCESSORS_Actual}    characters="
+    Should Be Equal  ${MM_PAYLOAD_PROCESSORS_Actual.strip()}  ${MM_PAYLOAD_PROCESSORS_Expected.strip()}
```

Comment:
Maybe we should also check if the pods are running e.g. `oc wait --for=condition=ready pod/<pod-name> --timeout=<duration>` or similar

Reply:
Done in line 36 now.